backend/endpoints/file.py

```diff
@@ -0,0 +1,109 @@
+from fastapi import  HTTPException, status
+from schema.response import ResponseModel
+from starlette.responses import JSONResponse, Response
+from fastapi import FastAPI, UploadFile, File, APIRouter
+from fastapi.responses import FileResponse, JSONResponse
+from utils.message_utils import get_room_messages
+import os
+import zipfile
+import io
+from zipfile import ZipFile
+router = APIRouter()
+
+"""
+ An get method endpoint to Handle retrieving and Viewing of uploaded files in a DM
+
+"""
+
+# {
+#     "organization_id" :"6133c5a68006324323416896",
+#     "room_id" : "6133c5a68006324323416896"
+
+        #         org_id:
+        # 6373eb474746182adae97314
+        # room_id:
+        # 6373eb4f4746182adae97316
+
+        #  Use:
+        # org_id: 619ba4
+        # room_id: 123456
+# }
+
+"""
+    A helper function to handle Multiple Uploaded image files in a DM
+
+"""
+
+def zipfiles(filenames):
+    """
+     Receives a list of  uploaded files
+     Returns:
+        An Archive file object
+    """
+    zip_filename = "archive.zip"
+
+    data = io.BytesIO()
+    Zipfile = zipfile.ZipFile(data, "w")
+
+    for fpath in filenames:
+        fdir, fname = os.path.split(fpath)
+        print(fname,fdir)
+        Zipfile.write(fpath, fname)
+    Zipfile.close()
+    resp = Response(data.getvalue(), media_type="image/png", headers={
+        'Content-Disposition': f'attachment;filename={zip_filename}'
+    })
+    return resp
+
+
+"""
+A helper asyn function to handle single Uploaded image file in a DM
+
+"""
+
+
+async def get_image(img: str):
+
+    if img:
+        return FileResponse("image.png", media_type="image/png")
+    else:
+        return {"file": "No file uploaded"}
+
+
+
+""""
+An Endpoint that handles viewing image files uploaded in the DM
+
+params:
+    org_id = Organisation id of user
+    room_id = room id of user
+
+Retrieves All Messages in the Dm with the get_room_message method
+Handles  conditons for single and Multiple Uploaded Files
+"""
+@router.get("/get/files/{org_id}/{room_id}")
```

Comment:
```suggestion
@router.get("/org/{org_id}/rooms/{room_id}/files")
```
your endpoint should look like this instead